frontends/concrete-python/examples/xor_distance/xor_distance.md

```diff
@@ -151,3 +156,35 @@ And finally, for 12804-bit vectors, execution times should be:
     dist_in_fhe_with_multivariate_tables on 4 bits: 40.89 seconds
 ```
 
+## Distance Between Two Encrypted Tensors
```

Comment:
This probably supposed to be:
```
## Distance Between An Encrypted Tensor And A Clear Tensor
```
Or something like that.

Reply:
Indeed!

Reply:
Done